src/highlight/HighlightAnnotations.scss

```diff
@@ -0,0 +1,9 @@
+.ba-HighlightAnnotations-creator {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    pointer-events: auto;
```

Comment:
Probably leave these out for now until we know what kind of event handling we'll need.

Reply:
Removed `touch-action`, but `pointer-events: auto` must be here to override `pointer-events: none` in `.ba-Layer`.